packages/discussions/src/types.ts

```diff
@@ -0,0 +1,831 @@
+import { IAuthenticationManager } from "@esri/arcgis-rest-request";
+// import { Geometry } from "geojson";
+
+/**
+ * sort orders
+ *
+ * @export
+ * @enum {number}
+ */
+export enum SortOrder {
+  ASC = "ASC",
+  DESC = "DESC"
+}
+
+/**
+ * reactions to posts
+ *
+ * @export
+ * @enum {number}
+ */
+export enum PostReaction {
+  THUMBS_UP = "thumbs_up",
+  THUMBS_DOWN = "thumbs_down",
+  THINKING = "thinking",
+  HEART = "heart",
+  ONE_HUNDRED = "one_hundred",
+  SAD = "sad",
+  LAUGH = "laugh",
+  SURPRISED = "surprised"
```

Comment:
since these are comments, just use the UTF8 emoji character 👍, 👎, etc.

Reply:
I'm sure it would work just fine, but since these values are ultimately persisted in the database I figure using human-readable string versions would eliminate any potential for things to go wrong if and when we choose to optimize by migrating these string values to numeric